.NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/Constants.cs

```diff
@@ -33,6 +33,14 @@ public static class Constants
         public const string HourUnit = "hour";
         public const string MinuteUnit = "minute";
         public const string SecondUnit = "second";
+        public const string TimeDurationUnit = "s";
+
+        public const string Now = "now";
```

Comment:
Are these general terms? "now" and "every"? If English specific, they should be resources or constants in an English file.